conans/client/generators/cmake_common.py

```diff
@@ -24,6 +26,11 @@
 set(CONAN_CXX_FLAGS_{dep}{build_type}_LIST "{deps.cxxflags_list}")
 set(CONAN_SHARED_LINKER_FLAGS_{dep}{build_type}_LIST "{deps.sharedlinkflags_list}")
 set(CONAN_EXE_LINKER_FLAGS_{dep}{build_type}_LIST "{deps.exelinkflags_list}")
+
+# Apple Frameworks
+conan_find_apple_frameworks(CONAN_FRAMEWORKS_{dep}{build_type} CONAN_FRAMEWORKS_FOUND_{dep}{build_type})
+# Append to aggregated values variable
+set(CONAN_LIBS_{dep}{build_type} ${{CONAN_LIBS_{dep}{build_type}}} ${{CONAN_SYSTEM_LIBS_{dep}{build_type}}} ${{CONAN_FRAMEWORKS_FOUND_{dep}{build_type}}})
```

Comment:
Wouldn't it be more clear if this aggregates  CONAN_PKG_LIBS instead of CONAN_LIBS? They have the same value above. Maybe even avoid the first definition.

Reply:
Yes, the variable can be changed to be more clear. But I don't want to change the previous definition of CONAN_LIBS as many test check the values there and it would require a lot of changes for this PR